packages/plugin-node-surrounding-code/surrounding-code.js

```diff
@@ -86,7 +86,7 @@ class CodeRange extends Writable {
     this._n++
     if (this._n < this._start) return cb(null)
     if (this._n <= this._end) {
-      this._code[String(this._n)] = chunk
```

Comment:
Is the substring centered on the column provided if we get one or is it just the start of line?

Reply:
It's just the start of the line. We could center around the column but only for one of the lines.

Reply:
yeah - might get confusing. ok!